### ISSUE TITLE
Incorrect use of the word convenience in log file viewer

### ISSUE THREAD

**Issue by Jordie0608** [nfreader/newSS13tools#68]
![1](https://user-images.githubusercontent.com/4343468/31646172-fc7b9ed8-b34b-11e7-97b5-5c5f6f93269e.PNG)
![1 - copy](https://user-images.githubusercontent.com/4343468/31646188-18167b68-b34c-11e7-83ce-291c5c0021df.PNG)
![2](https://user-images.githubusercontent.com/4343468/31646174-feca6d72-b34b-11e7-947b-532a7f072410.PNG)
![1 - copy 2](https://user-images.githubusercontent.com/4343468/31646199-264fc770-b34c-11e7-8767-80eedee7688e.PNG)

Also the lines being split into columns is quite annoying for some of these, especially for the runtime log.

**Comment by nfreader** [comment]
Yes. The log viewer is a WIP. What you’re seeing is the pda log being parsed by the generic parsing method. I’ll be going through and adding more parsers over the next few days.
